README.md

```diff
@@ -29,6 +29,10 @@ $ make
 This will produce object files for all the classes as well as the main binary `fasttext`.
 If you do not plan on using the default system-wide compiler, update the two macros defined at the beginning of the Makefile (CC and INCLUDES).
 
+### Building with Docker
+
+To build with Docker, see [fastText-Docker](https://github.com/xeb/fastText-docker) repository.
```

Comment:
I would rather be less directive and simply advertise that a docker build file is available at.
Something like: "If you want to build with docker, please check ...".
Thanks!